src/current/_includes/common/license-expiration-messages.md

```diff
@@ -0,0 +1,9 @@
+Depending on the status of your cluster's [license]({% link {{ page.version.version }}/licensing-faqs.md %}), the following messages may be displayed in the DB Console:
+
+- If the license will expire soon, the message is: _This cluster will require a license key by ${DATE} or the cluster will be throttled._
```

Comment:
This error is also displayed if there is no license key in the first place.

Reply:
We don't actually display a message when the cluster is expiring soon. We only display the banner alerts when the license has actually expired and the cluster is in a grace period before clusters are throttled: 

"Your license key expired on <date>. The cluster will be throttled on <date> unless a new license key is added. Learn more"

Reply:
> I believe this error is only displayed if there is no license key in the first place (not is expiring soon)
> We don't actually display a message when the cluster is expiring soon.

Ah ok, i took these from some UI screenshots that were shared a while back.  Makes sense that they got out of date, thanks for the updated info

> We only display the banner alerts when the license has actually expired and the cluster is in a grace period before clusters are throttled:

> "Your license key expired on . The cluster will be throttled on unless a new license key is added. Learn more"

Ok, updated to the following:

- If there is no valid license key and the cluster is still in the grace period, the message is: _Your license key expired on ${DATE1}. The cluster will be throttled on ${DATE2} unless a new license key is added._

Reply:
@jaiayu OK with tehse changes? OK to resolve this convo?